src/main/java/org/gridsuite/study/server/repository/StudyEntity.java

```diff
@@ -119,13 +117,8 @@ public class StudyEntity extends AbstractManuallyAssignedIdentifierEntity<UUID>
     @Column(name = "securityAnalysisParametersUuid")
     private UUID securityAnalysisParametersUuid;
 
-    @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
-    @JoinColumn(name = "sensitivityAnalysisParametersEntity_id",
-            referencedColumnName = "id",
-            foreignKey = @ForeignKey(
-                    name = "sensitivityAnalysisParameters_id_fk"
-            ))
-    private SensitivityAnalysisParametersEntity sensitivityAnalysisParameters;
+    @Column(name = "sensitivityAnalysisParametersUuid")
+    private UUID sensitivityAnalysisParametersUuid;
```

Comment:
We must keep the old tables until the data migration is done
```suggestion
    /**
     * @deprecated to remove when the data is migrated into the sensitivity-analysis-server
     */
    @Deprecated(forRemoval = true)
    @Getter(AccessLevel.PROTECTED)
    @Setter(AccessLevel.PROTECTED)
    @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
    @JoinColumn(name = "sensitivityAnalysisParametersEntity_id",
            referencedColumnName = "id",
            foreignKey = @ForeignKey(
                    name = "sensitivityAnalysisParameters_id_fk"
            ))
    private SensitivityAnalysisParametersEntity sensitivityAnalysisParameters;
    
    @Column(name = "sensitivityAnalysisParametersUuid")
    private UUID sensitivityAnalysisParametersUuid;
```

Reply:
Done